MainModule.ioc

```diff
@@ -56,7 +56,7 @@ Mcu.Pin37=PD5
 Mcu.Pin38=PB3
 Mcu.Pin39=PB6
 Mcu.Pin4=PH0-OSC_IN
-Mcu.Pin40=PB9
+Mcu.Pin40=PB7
```

Comment:
Why is this Mcu Pin changing values? Will this cause compliance issues with the existing Main Module implementation?